src/package.json

```diff
@@ -30,7 +30,7 @@
     "@react-native-community/eslint-config": "0.0.5",
     "babel-jest": "^24.9.0",
     "babel-plugin-module-resolver": "3.1.3",
-    "detox": "12.4.1",
```

Comment:
I updated the `detox` package because the installation process was failing due to incompatibilities with newer Xcode versions.